mantidproject/mslice
# Fix AttributeError when loading cut

Loading a cut workspace used to cause an AttributeError, now it loads correctly. 

**To test:**
- Load file, calculate projections, input cut parameters
- Save cut to workspace
- Save cut to file
- Load cut

Loading should succeed, and the cut should be plottable (as long as parent workspace is in ADS - see #51 )

Fixes #146
